### PR TITLE
feat(496): add stream links

### LINKED ISSUE #496: Add stream links in agenda and navbar
day 1
alpha https://youtube.com/live/_mA4mvYpU68
beta https://youtube.com/live/te8tNwnej0M
day 2
alpha https://youtube.com/live/8owI4xBEIl0
beta https://youtube.com/live/Va6vIYCVxj0

### PR DESCRIPTION
**Pull request by TheJoin95**
Closes #496 

image

### CODE REVIEW

File: pages/agenda.tsx [hunk 1]
```diff
@@ -53,6 +54,14 @@ export default function Agenda() {
           </div>
         </div>
 
+        {new Date().getTime() > 1709798400000 && 
+        <div className="container-stream-link">
+          <h2>We are live!</h2>
+          <a className="button" href={YT_LIVE_LINKS[trackSelected.toLowerCase()]} rel="nofollow" target="_blank">
```
Comment (Wabri) [inline pages/agenda.tsx]:
```
4:17:35 PM: 60:11  Error: Using target="_blank" without rel="noreferrer" (which implies rel="noopener") is a security risk in older browsers: see https://mathiasbynens.github.io/rel-noopener/#recommendations  react/jsx-no-target-blank
```

Maybe without the `rel=...`???
```suggestion
          <a className="button" href={YT_LIVE_LINKS[trackSelected.toLowerCase()]} target="_blank">
```

**Review by Wabri — CHANGES_REQUESTED** [review]

File: pages/agenda.tsx [hunk 2]
```diff
@@ -53,6 +54,14 @@ export default function Agenda() {
           </div>
         </div>
 
+        {new Date().getTime() > 1709798400000 && 
+        <div className="container-stream-link">
+          <h2>We are live!</h2>
+          <a className="button" href={YT_LIVE_LINKS[trackSelected.toLowerCase()]} target="_blank">
```
Comment (davideimola) [inline pages/agenda.tsx]:
```suggestion
          <a className="button" href={YT_LIVE_LINKS[trackSelected.toLowerCase()]} target="_blank" rel="noreferrer">
```

**Review by davideimola — CHANGES_REQUESTED** [review]

**Comment by Wabri** [comment]
```
3:57:26 PM: ./pages/agenda.tsx:60:39
3:57:26 PM: Type error: Element implicitly has an "any" type because expression of type "string" can"t be used to index type "{ alpha1: string; alpha2: string; beta1: string; beta2: string; }".
  No index signature with a parameter of type "string" was found on type "{ alpha1: string; alpha2: string; beta1: string; beta2: string; }".
3:57:26 PM:   58 |         <div className="container-stream-link">
3:57:26 PM:   59 |           <h2>We are live!</h2>
3:57:26 PM: > 60 |           <a className="button" href={YT_LIVE_LINKS[trackSelected.toLowerCase()]} target="_blank" rel="noreferrer">
3:57:26 PM:      |                                       ^
3:57:26 PM:   61 |             Click to follow the live stream on YouTube!
3:57:26 PM:   62 |           </a>
3:57:26 PM:   63 |         </div>}
```

**Comment by TheJoin95** [comment]
@Wabri looking good now

**Review by davideimola — APPROVED** [review]

**Review by Wabri — CHANGES_REQUESTED** [review]
Yes it build, but there are no links show!
Screenshot 2024-03-02 at 18 21 03

**Comment by TheJoin95** [comment]
@Wabri please read the code, this is a time enabling feature

**Comment by Wabri** [comment]
@TheJoin95 there is no time to do something like that 😁 sorry I'll accept and bow until I feel pain in my back!

**Review by Wabri — APPROVED** [review]

**Comment by TheJoin95** [comment]
@Wabri no pain, no gain

**Pull request merged**